templates/partials/question.html

```diff
@@ -20,6 +20,13 @@
   {%- endif -%}
 {%- endset -%}
 
+{% set individual_response_guidance %}
+   {%- if show_individual_response_guidance == True -%}
+      {% set title = _("If you can’t answer questions for this person") %}
+      {% include 'partials/individual-response-guidance.html' %}
+   {%- endif -%}
+{% endset %}
```

Comment:
Should this link go to the "Change" page if they have already requested an IR because that is what we do from the hub. Worth clarifying. Otherwise, works as expected. 👍

Reply:
Confirmed the current journey with @donnap573